<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>429</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    font: bold;
	font-size:16px;
    border: 1px solid silver;
    border-radius: 12px;
    margin-top: 8px;
    padding: 15px 5px 10px 5px;
background-color: rgb(250, 250, 250);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
}


QPushButton {
	color: white;
    border-width: 2px;
    border-color: beige;
}

</string>
   </property>
   <property name="title">
    <string>Select Role</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_retailer">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="retailer_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(67,102,48);
    border-style: outset;
    border-radius: 10px;
    font: bold 12px;
    padding: 6px;
	color: white;
}
#retailer_btn:hover{ 
    background-color: rgb(106,208,116);
background-color: qlineargradient(spread:pad, x1:0, y1:0.005, x2:1, y2:1, stop:0 rgba(66, 163, 167, 255), stop:1 rgba(76, 92, 171, 255));
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
	color: white;}

#retailer_btn:disabled {
	background-color: #add8e6;
	color: black;
}
</string>
         </property>
         <property name="text">
          <string>Retailer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_wholesaler">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wholesaler_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(55,69,156);
    border-style: outset;
    border-radius: 10px;
    font: bold 12px;
    padding: 6px;
color: white;
}

#wholesaler_btn:hover{ 
    background-color: rgb(106,208,116);
background-color: qlineargradient(spread:pad, x1:0, y1:0.005, x2:1, y2:1, stop:0 rgba(66, 163, 167, 255), stop:1 rgba(76, 92, 171, 255));
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
	color: white;}

#wholesaler_btn:disabled {
	background-color: #add8e6;
	color: black;
}
</string>
         </property>
         <property name="text">
          <string>Wholesaler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_distributor">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="distributor_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(207,163,43);
    border-style: outset;
    border-radius: 10px;
    font: bold 12px;
    padding: 6px;
color: white;
}

#distributor_btn:hover{ 
    background-color: rgb(106,208,116);
background-color: qlineargradient(spread:pad, x1:0, y1:0.005, x2:1, y2:1, stop:0 rgba(66, 163, 167, 255), stop:1 rgba(76, 92, 171, 255));
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
	color: white;}

#distributor_btn:disabled {
	background-color: #add8e6;
	color: black;
}
</string>
         </property>
         <property name="text">
          <string>Distributor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_factory">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="factory_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(240,70,80);
    border-style: outset;
    border-radius: 10px;
    font: bold 12px;
    padding: 6px;
color: white;
}

#factory_btn:hover{ 
    background-color: rgb(106,208,116);
background-color: qlineargradient(spread:pad, x1:0, y1:0.005, x2:1, y2:1, stop:0 rgba(66, 163, 167, 255), stop:1 rgba(76, 92, 171, 255));
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
	color: white;}

#factory_btn:disabled {
	background-color: #add8e6;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>Factory</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
